cli/simulate_pixels.py

```diff
@@ -94,6 +95,13 @@ def run_simulation(input_filename,
             input_has_trajectories = True
         except KeyError:
             input_has_trajectories = False
+    
+    # Made a separate array to store number of photons generated from quenching.py. Should be added to tracks as another dtype
+    phot_from_edep = np.zeros(len(tracks), dtype = [('n_photons_edep','f4')])
```

Comment:
I would rather avoid creating a new array containing just another field. Could you add a line to `dumpTree.py` that creates this column? In order to use also existing h5 files, you could check if `n_photons_edep` is present, and if not then you can create a copy of `tracks` with this extra column. I prefer this solution than having an independent array. Also, can we just call it `n_photons`? Or is it confusing? I don't have a strong preference.

Reply:
Fixed in https://github.com/DUNE/larnd-sim/pull/39/commits/5bee7527634b636be39c7c0f23f67f47bcd0c0ea . However I should mention that we weren't actually able to test this by rerunning `dumpTree.py`,  so you may want to validate this yourself just to be sure things are working as they should.  Also, I do agree with calling it `n_photons`, it makes more sense in the context of the rest of `tracks` and it should be pretty easy to distinguish between `n_photons` and `n_photons_det` as they're each in their own separate dataset .